samples/multicluster/expose-istiod.yaml

```diff
@@ -8,15 +8,15 @@ spec:
     istio: eastwestgateway
   servers:
     - port:
-        name: tcp-istiod
+        name: https-istiod
         number: 15012
-        protocol: TCP
```

Comment:
I don't think this is valid without TLS settings configured

Reply:
yes, thanks for the reminder!  expose-istiod.yaml is used in our testing as well.  I've updated it to do passthrough vs terminate then disable tls, hope all tests can pass.   I also added a sample for expose istiod as https so we can use it in our docs later on.